.github/workflows/codeowners.yml

```
@@ -0,0 +1,19 @@
+name: Ping maintainers
+
+on:
+  pull_request:
+    types: [opened, synchronize, ready_for_review]
+  push:
+    types: [opened, synchronize, ready_for_review]
+
+jobs:
+  codenotify:
+    runs-on: ubuntu-latest
```

Same as is set in the other actions, so it doesn't request reviews in peoples forks.

```suggestion
    runs-on: ubuntu-latest
    if: github.repository_owner == 'NixOS'
```